webextensions/api/webRequest.json

```diff
@@ -117,18 +117,29 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/RequestFilter",
             "support": {
               "chrome": {
+                "notes": "If a filter contains unrecognized values in its <code>types</code> property, this is treated as an error.",
```

Comment:
I'm sorry I'm familiar enough with this API to provide a more direct suggestion, but can we be more explicit about what "treated as an error" means? Something like

* If a filter contains unrecognized values in its <code>types</code> property, then the callback throws an exception.
* …, then the `addListener` method returns an error value.

Or whatever makes sense in this case.